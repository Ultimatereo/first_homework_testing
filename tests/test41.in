5
5 2 4 1 4 0
5 2 3 4 3 5
1 4 5 2 0 5
4 4 0 3 0 3
0 1 1 5 5 3
