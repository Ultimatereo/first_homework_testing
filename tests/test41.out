0.101112
-0.537851
0.777131
1.58232
-1.03017
